### PR TITLE
[BFN] Implement platform API for thermal

### PR DESCRIPTION
**Pull request by VadymYashchenko**
Signed-off-by: Vadym Yashchenko <vadymx.yashchenko@intel.com>

**What I did**
	

- Add get_model() function

	

- Add get_low_critical_threshold() function

	

- Change __get(...) function.

**How I did it**
	

- Difference from previous implementation of __get(...) function is return real value or -9999.9 if value is not provided by thrift API

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py [hunk 1]
```diff
@@ -76,18 +76,27 @@ def __init__(self, chip, label):
...
 
     # ThermalBase interface methods:
     def get_temperature(self) -> float:
         return float(self.__get('temp', 'input'))
-
+ 
     def get_high_threshold(self) -> float:
         return float(self.__get('temp', 'max'))
 
+
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py:91]: please remove extra line

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py [hunk 2]
```diff
@@ -76,18 +76,27 @@ def __init__(self, chip, label):
...
+        if value is not None: 
+            return value
+        else:
+            return -999.9
 
     # ThermalBase interface methods:
     def get_temperature(self) -> float:
         return float(self.__get('temp', 'input'))
-
+
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py:87]: please remove whitespace

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py [hunk 3]
```diff
@@ -76,18 +76,27 @@ def __init__(self, chip, label):
     def __get(self, attr_prefix, attr_suffix):
         sensor_data = _sensors_get().get(self.__chip, {}).get(self.__label, {})
         value = _value_get(sensor_data, attr_prefix, attr_suffix)
-        if value is not None: return value
-        raise NotImplementedError
+        if value is not None:
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py:79]: Please update as follows:
```
    return value if value is not None else -999.9
```

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py (hunk 1 above)
Comment (VadymYashchenko) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py]: done

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py (hunk 3 above)
Comment (VadymYashchenko) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py]: done

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py (hunk 2 above)
Comment (VadymYashchenko) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py]: done

**Review by akokhan — APPROVED** [review]

**Comment by VadymYashchenko** [comment]
Already merged

**Pull request closed without merging**